src/Fmt_ast.ml

```diff
@@ -2938,6 +2938,13 @@ and fmt_extension_constructor c sep ctx ec =
        $ fmt_attributes c ~pre:(fmt "@ ") ~key:"@" atrs ~suf
        $ fmt_docstring_padded c doc )
 
+and fmt_functor_arg c (name, mt) =
+  let fmt_mty mt =
+    let mty = fmt_module_type c mt in
+    compose_module mty ~f:(fun k -> fmt " : " $ k)
```

Comment:
Maybe `"@ : "` in case of long name ?

Reply:
Fixed, and no diff with test_branch by the way.